hacking/aws_config/testing_policies/database-policy.json

```diff
@@ -78,6 +78,17 @@
           ],
           "Effect": "Allow",
           "Resource": "*"
+        },
+        {
+          "Sid": "DMSEndpoints",
+          "Effect": "Allow",
+          "Action": [
+            "dms:CreateEndpoint",
```

Comment:
Were these intended to be in this PR?

Reply:
I'm not really sure how they came to be in this PR but given that there are DMS tests, and these are the only dms privileges in the testing policies, I'm favouring leaving them in

Reply:
This would eliminate the need for the policy creation in #56980 then, right?

Reply:
@jillr yes, they wouldn't be needed there